packages/env-utils/src/envUtils.native.ts

```diff
@@ -24,6 +25,10 @@ const getDeviceType = () => '';
 
 const getOsVersion = () => `${Platform.Version}`;
 
+const getSuiteVersion = () => Config.VERSION || '';
```

Comment:
We can remove these functions from `suite-native/config/src/debugUtils.ts`

Reply:
like this? ec280816507d7c2a71db721fad8462fc5b19b09c

Reply:
nice